pkg/graph/schema.graphql

```diff
@@ -2036,17 +2040,18 @@ enum ConfidenceType {
 #   OTHER
 # }
 
-enum DataFrequencyType {
-    ANNUALLY
-    BIANNUALLY
-    QUARTERLY
-    MONTHLY
-    SEMI_MONTHLY
-    WEEKLY
-    DAILY
-    OTHER
-    NOT_PLANNING_TO_DO_THIS
-}
+
+# enum DataFrequencyType {
+#   ANNUALLY
+#   BIANNUALLY
+#   QUARTERLY
+#   MONTHLY
+#   SEMI_MONTHLY
+#   WEEKLY
+#   DAILY
+#   OTHER
+#   NOT_PLANNING_TO_DO_THIS
+# }

```

Comment:
leftover commented code

Reply:
We are leaving that in intentionally for Toms work